.goreleaser.yml

```diff
@@ -0,0 +1,132 @@
+before:
+  hooks:
+    - go version | grep --quiet "go1\.15\.2" || echo "Go binary version must be 1.15.2"
+    - go mod download
+builds:
+  - id: operator-sdk
+    main: ./cmd/operator-sdk
+    binary: operator-sdk
+    env:
+      - CGO_ENABLED=0
+      - GO111MODULE=on
+      - REPO=github.com/operator-framework/operator-sdk
```

Comment:
Should we also explicitly set `GOPROXY=https://proxy.golang.com,direct`? We've seen some issues where different distros set this in different ways. It would be more foolproof if we are explicit.

Reply:
We can go further with the new [pipe syntax](https://golang.org/doc/go1.15#go-command): `GOPROXY=https://proxy.golang.com|direct`.